src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
```

Comment:
:information_source: Unnecessary field declaration outside the class. (Severity: Minor)